foreach1.js

```diff
@@ -9,3 +9,6 @@
 var foods = ["pizza", "tacos", "ice cream", "sushi"];
 
 // your code here
+foods.forEach((element) => {
+    console.log('I like', element)
+    })
```

Comment:
nice! your closing brackets/parentheses should be at the same level as the line where their buddies are, though. Small indentation detail, but it's worth noting